### PR TITLE
[chassis][voq] remove conditional mark for voq tests

### LINKED ISSUE #6171: [chassis] - voq_ipfwd tests are getting skipped
**Description**
PR# https://github.com/sonic-net/sonic-mgmt/pull/5708/ introduced using conditional marks instead of pytest.skip in the test scripts.
However, with this PR that is causing all tests voq/test_voq_ipfwd.py to be skipped on T2 chassis.  The are failing with the reason
```
Did not find any asic in the DUTs (linecards) that are connected to T1 VM's
```

We see the following issues:

- mg_facts are loaded in [init.py ](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/plugins/conditional_mark/__init__.py#L311) for the first DUT in the testbed.  If the first DUT is connected to T3 VM's, then the tests are going to skip, even though we have another linecard that has T1 VM's connected.
   - The original voq_ipfwdtest was checking minigraph_neighbors on all the linecards (DUTs) in the chassis.
- For a multi-asic card, VOQ_INBAND_INTERFACE and BGP_VOQ_CHASSIS_NEIGHBOR are defined in asic config db's and not in the global config_db.json. Thus, these fields are not populated correctly in the facts gathered in __init__.py in conditional mark - as the config facts are not taking into consideration all the asics.

**Steps to reproduce the issue:**
1. Run voq_ipfwd.py suite against a T2 chassis with multi-asic linecards and having the first DUT be the linecard that connects to only T3 VM's.
2.
3.

**Describe the results you received:**
Tests are getting skipped.

**Describe the results you expected:**
Tests should run and not get skipped.

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

### PR DESCRIPTION
**Pull request by arlakshm**
Signed-off-by: Arvindsrinivasan Lakshmi Narasimhan <arlakshm@microsoft.com>

<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/sonic-net/SONiC/blob/gh-pages/CONTRIBUTING.md

Please provide following information to help code review process a bit easier:
-->
### Description of PR
<!--
- Please include a summary of the change and which issue is fixed.
- Please also include relevant motivation and context. Where should reviewer start? background context?
- List any dependencies that are required for this change.
-->

Summary:
Fixes #6171 


### Type of change

<!--
- Fill x for your type of change.
- e.g.
- [x] Bug fix
-->

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)


### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

### Approach
#### What is the motivation for this PR?
The PR https://github.com/sonic-net/sonic-mgmt/pull/5708 causes  the tests in the files `test_voq_nbr.py` and `test_voq_ipfwd.py` to be skipped. 


#### How did you do it?
This PR removes the voq test from condional mark and revert the the changes done in PR  for these 2 files in PR #5708 
#### How did you verify/test it?
Run voq tests
```
arlakshm@681605572cc8:/data/my-mgmt/tests$ sudo ANSIBLE_KEEP_REMOTE_FILES=1  ./run_tests.sh -c 'voq/test_voq_ipfwd.py::TestTableValidation::test_host_route_table_local_addr[str2-sonic-lc5-1-0]'  -i '../ansible/str2,../ansible/veos' -n
 'vms26-t2-sonic-1' -t 't2,any' -e '--skip_sanity'  -u                                                                                                                                                                                    
=== Running tests in groups ===                                                                                                                                                                                                          
Running: pytest voq/test_voq_ipfwd.py::TestTableValidation::test_host_route_table_local_addr[str2-sonic-lc5-1-0] --inventory ../ansible/str2,../ansible/veos --host-pattern str2-sonic-lc5-1,str2-sonic-lc6-1,str2-sonic-lc7-1,str2-sonic-sup-
1 --testbed vms26-t2-sonic-1 --testbed_file /data/my-mgmt/ansible/testbed.yaml --log-cli-level warning --log-file-level debug --kube_master unset --showlocals --assert plain --show-capture no -rav --allow_recover --ignore=ptftests --i
gnore=acstests --ignore=saitests --ignore=scripts --ignore=k8s --ignore=sai_qualify --junit-xml=logs/tr.xml --log-file=logs/test.log --topology t2,any --skip_sanity                                                                     
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be remov
ed in the next release.                                                                                                                                                                                                                  
  from cryptography.exceptions import InvalidSignature                                                                                                                                                                                   
========================================================================================================== test session starts ==========================================================================================================
platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.11.0, pluggy-0.13.1                                                                                                                                                                 
ansible: 2.8.12                                                                                                                                                                                                                          
rootdir: /data/my-mgmt/tests, inifile: pytest.ini                                                                                                                                                                                        
plugins: html-1.22.1, repeat-0.9.1, forked-1.3.0, xdist-1.28.0, metadata-1.11.0, allure-pytest-2.8.22, ansible-2.2.2                                                                                                                     
collecting ...                                                                                                                                                                                                                           
---------------------------------------------------------------------------------------------------------- live log collection ----------------------------------------------------------------------------------------------------------
05:33:11 testbed.get_testbed_type                 L0266 WARNING| Unsupported testbed type - appliance                                                                                                                                    
05:33:11 testbed.get_testbed_type                 L0266 WARNING| Unsupported testbed type - appliance                                                                                                                                    
05:33:27 testbed.get_testbed_type                 L0266 WARNING| Unsupported testbed type - appliance                                                                                                                                    
05:33:27 testbed.get_testbed_type                 L0266 WARNING| Unsupported testbed type - appliance                                                                                                                                    
collected 1 item                                                                                                                                                                                                                         
                                                                                                                                                                                                                                         
voq/test_voq_ipfwd.py::TestTableValidation::test_host_route_table_local_addr[str2-sonic-lc5-1-0] PASSED                                   
```
#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
<!--
(If it's a new feature, new test case)
Did you update documentation/Wiki relevant to your implementation?
Link to the wiki page?
-->

### CODE REVIEW

**Comment by arlakshm** [comment]
@ysmanman @sanmalho-git  for viz.

**Comment by sanmalho-git** [comment]
@arlakshm we need similar change for snmp tests (test_snmp_interfaces, test_snmp_phy_entity, and test_snmp_queue.py) which check for 'is_supervisor'

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -806,35 +806,11 @@ show_techsupport/test_techsupport.py::test_techsupport:
...
-  skip:
-    reason: "LLDP not supported on supervisor node."
-    conditions:
-      - "is_supervisor"
-
-snmp/test_snmp_pfc_counters.py:
-  skip:
-    reason: "M0/MX topo does not support test_snmp_pfc_counters"
-    conditions:
-      - "topo_name in ['m0', 'mx']"
```
Comment (wangxin) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:825]: Why remove this skipping for m0 and mx topology?

File: tests/snmp/test_snmp_phy_entity.py [hunk 2]
```diff
@@ -344,6 +344,8 @@ def test_psu_info(duthosts, enum_rand_one_per_hwsku_hostname, snmp_physical_enti
     """
     snmp_physical_entity_info = snmp_physical_entity_and_sensor_info["entity_mib"]
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    if not duthost.is_supervisor_node():
+        pytest.skip("Not supported on non supervisor node")
```
Comment (SuvarnaMeenakshi) [inline tests/snmp/test_snmp_phy_entity.py:348]: This test is getting skipped based on the check below which looks for PSU_INFO in state db.
Do we need another skip here?
The current skip will look like this: "Fan information does not exist in DB, skipping this test"

File: tests/snmp/test_snmp_lldp.py [hunk 3]
```diff
@@ -35,6 +37,8 @@ def test_snmp_lldp(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds_
     (similar to lldp test)
     """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    if duthost.is_supervisor_node():
+        pytest.skip("LLDP not supported on supervisor node")
```
Comment (SuvarnaMeenakshi) [inline tests/snmp/test_snmp_lldp.py:41]: We need lldp checks for supervisor as well.
The below OIDs needs to tested.
lldpLocChassisIdSubtype', 'lldpLocChassisId', 'lldpLocSysName', 'lldpLocSysDesc'
'lldpLocManAddrLen', \
'lldpLocManAddrIfSubtype', \
'lldpLocManAddrIfId', \
'lldpLocManAddrOID']:

The checks related snmp_facts['snmp_interfaces'] are not required, but test case already handles that check as it compares with minigraph neighbor information and we dont have any neighbors for supervisor.

File: tests/snmp/test_snmp_phy_entity.py (hunk 2 above)
Comment (arlakshm) [reply tests/snmp/test_snmp_phy_entity.py:348]: Hi @SuvarnaMeenakshi , this PR is undoing an earlier change which causes all these tests to skip. we can add more checks later on top of thise

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml (hunk 1 above)
Comment (arlakshm) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:877]: remove this in the latest commit

**Review by wangxin — APPROVED** [review]

**Pull request merged**